src/base.jl

```diff
@@ -44,30 +46,44 @@ struct PowerSystem{L <: ElectricLoad,
     branches::B
     storage::S
     basepower::Float64 # [MVA]
-    time_periods::Int64
+    forecasts::F
+    services::V
+    annex::Union{Nothing,Dict{Any,Any}}
 
     function PowerSystem(buses::Array{Bus,1},
                         generators::Array{G,1},
                         loads::Array{L,1},
                         branches::Nothing,
                         storage::Nothing,
-                        basepower::Float64; kwargs...) where {G <: Generator, L <: ElectricLoad}
+                        basepower::Float64,
+                        forecasts::F,
+                        services::V,
+                        annex::Union{Nothing,Dict{Any,Any}}; kwargs...) where {G <: Generator, 
```

Comment:
So how you do feel about complying with our new coding standards about whitespace, particularly around commas?  :)